demos/chat-client/c++/process_assets.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nix-shell
```

Comment:
We should not make `nix-shell` a dependency here
```suggestion
#!/usr/bin/env python
```